src/main/java/org/codeforamerica/shiba/output/caf/HourlyJobIncomeInformation.java

```diff
@@ -23,7 +27,7 @@ public boolean isComplete() {
     }
 
     @Override
-    public Double grossMonthlyIncome() {
-        return hourlyWage * hoursAWeek * 4;
+    public Money grossMonthlyIncome() {
```

Comment:
what should this do when hourlywage or hours a week are null?

Reply:
That's handled in the GrossMonthlyIncomeParser